src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass.cs

```diff
@@ -1227,7 +1228,7 @@ protected virtual void VisitArguments(ImmutableArray<BoundExpression> arguments,
                 RefKind refKind = GetRefKind(refKindsOpt, i);
                 if (refKind != RefKind.Out)
                 {
-                    VisitRvalue(arguments[i]);
+                    VisitRvalue(arguments[i], isKnownToBeAnLvalue: refKind != RefKind.None);
```

Comment:
>RefKind.None); [](start = 78, length = 14)

how about RefKind.In?

Reply:
Those are not known to be an lvalue.

---
In reply to: [301813498](https://github.com/dotnet/roslyn/pull/37052#discussion_r301813498) [](ancestors = 301813498)